config/locales/nl_node_details.yml

```diff
@@ -0,0 +1,55 @@
+---
+nl:
```

Comment:
Would anyone have time to add translations for these attribute names in the technical specs table?

Reply:
Done!